apps/desktop/src/lib/forge/shared/prFooter.ts

```diff
@@ -33,18 +33,23 @@ export async function updatePrDescriptionTables(prService: ForgePrService, prNum
 /**
  * Replaces or inserts a new footer into an existing body of text.
  */
-function updateBody(body: string | undefined, prNumber: number, allPrNumbers: number[]) {
+function updateBody(
```

Comment:
Could we pass the symbol in here too, so that only `updatePrDescriptionTables` needs a ForgeService in order to be tested?